int32 pin
string type
int32 value
---
bool status

